apps/greenhouse/src/lib/plugin.js

```diff
@@ -83,7 +83,7 @@ const Plugin = ({ environment, apiEndpoint, currentHost }) => {
           id: "greenhouse-management",
           name: "greenhouse-management",
           displayName: "Organization",
-          version: managementVersion,
+          version: environment =='qa' || environment == 'dev' ? 'latest' : managementVersion, // pull latest version in dev and qa
```

Comment:
The same we here with the environment `dev` string

Reply:
:100: fixed